openreview/tools.py

```diff
@@ -703,3 +698,39 @@ def post_submission_groups(client, conference_id, submission_invite, chairs):
             readers=[conference_id, chairs],
             signatories=[]))
 
+def get_submission_invitations(client, status='all'):
+    '''
+    Returns a list of invitations ids according to the status.
+    
+    :arg client: Object of :class:`~openreview.Client` class
+    :arg status: Status filter for invitations. Default value is "all". This is a string with following possible value: all, open and closed.
+    '''
+    
+    try:
+        # Get a list of all invitations satisfying the regex
+        invitations = client.get_invitations(regex='.*/-/.*[sS]ubmission.*')
```

Comment:
We can filter the invitation in the server using the parameters: minduedate.
 
In this case the status parameter can be renamed to open_only: [True, False]

if open_only is true then send minduedate = now
if open_only is false or None then don't send minduedate parameter. 

Does it make sense?

Reply:
Suggested changes have been implemented.